pyannote/audio/interactive/v03.py

```diff
@@ -0,0 +1,193 @@
+        file = {"uri": text, "database": source, "audio": path}
+
+        duration = raw_audio.get_duration(file)
+        file["duration"] = duration
+
+        prodigy.log(f"RECIPE: detecting speech regions in '{path}'")
+
+        speech: Annotation = pipeline(file)
+
+        if duration <= chunk:
+            waveform, sr = raw_audio.crop(file, Segment(0, duration))
+            waveform = waveform.numpy()
+            task_audio = to_base64(normalize(waveform), sample_rate=SAMPLE_RATE)
+            task_audio_spans = to_audio_spans(speech)
+
+            yield {
+                "path": path,
+                "text": text,
+                "audio": task_audio,
+                "audio_spans": task_audio_spans,
+                "audio_spans_original": deepcopy(task_audio_spans),
+                "chunk": {"start": 0, "end": duration},
+                "meta": {"file": text},
+                # this is needed by other recipes
+                "recipe": "pyannote.sad.manual",
+            }
+
+        else:
+            for focus in chunks(duration, chunk=chunk, shuffle=False):
+                task_text = f"{text} [{focus.start:.1f}, {focus.end:.1f}]"
+                waveform, sr = raw_audio.crop(file, focus)
+                waveform = waveform.numpy().T
+                task_audio = to_base64(normalize(waveform), sample_rate=SAMPLE_RATE)
+                task_audio_spans = to_audio_spans(
+                    speech.crop(focus, mode="intersection"), focus=focus
+                )
+
+                yield {
+                    "path": path,
+                    "text": task_text,
+                    "audio": task_audio,
+                    "audio_spans": task_audio_spans,
+                    "audio_spans_original": deepcopy(task_audio_spans),
+                    "chunk": {"start": focus.start, "end": focus.end},
+                    "meta": {
+                        "file": text,
+                        "start": f"{focus.start:.1f}",
+                        "end": f"{focus.end:.1f}",
+                    },
+                    # this is needed by other recipes
+                    "recipe": "pyannote.sad.manual",
+                }
+
+
+@recipe(
+    "audio.v03",
+    dataset=("Dataset to save annotations to", "positional", None, str),
+    source=(
+        "Data to annotate (file path or '-' to read from standard input)",
+        "positional",
+        None,
+        str,
+    ),
+    chunk=(
+        "split long audio files into shorter chunks of that many seconds each",
+        "option",
+        None,
+        float,
+    ),
+)
+def v03(
+    dataset: str,
+    source: Union[str, Iterable[dict]],
+    loader: Optional[str] = "audio",
+    chunk: float = 10.0,
+    autoplay: bool = False,
+    keep_base64: bool = False,
+    fetch_media: bool = False,
+    exclude: Optional[List[str]] = None,
+) -> Dict[str, Any]:
+
+    log("RECIPE: Starting recipe audio.v03", locals())
+    label = ["Speech"]
```

Comment:
Since `label` is used only once, there is probably no need to create this variable.